app/Repositories/PhotoRepository.php

```diff
@@ -39,18 +40,22 @@ public function __construct(AWS $aws, Registrar $registrar)
      * @param  int $signupId
      * @return \Rogue\Models\Photo|null
      */
-    public function create(array $data, $signupId)
+    public function create(array $data, Signup $signup)
```

Comment:
@sbsmith86 I'm not sure the best way to do this part. I structured the request (up in the write up) so that photos are in their own array with all the data that lives on a `photo` and not a `signup`. Since `signup` needs `northstar_id` I kept that outside and it seemed weird to me to have it in each `photo` array as well. Although now I'm thinking it would be easier to just have `northstar_id` both outside for the signup and then with the data for each `photo` as well so the code doesn't need to be altered since this is just for the purposes of the migration anyway. Does that sound reasonable?